Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1742,6 +1742,13 @@ extension BrowserViewController: ContextMenuHelperDelegate {
             actionSheetController.addAction(copyAction)
         }
 
+        // If we're showing an arrow popup, set the anchor to the long press location.
+        if let popoverPresentationController = actionSheetController.popoverPresentationController {
+            popoverPresentationController.sourceView = view
+            popoverPresentationController.sourceRect = CGRect(origin: gestureRecognizer.locationInView(view), size: CGSizeMake(0, 16))
```

Comment:
Hm, the location looks correct to me when I test in the simulator. Where are you getting (-8, -8) from?